datadog/resource_datadog_child_organization.go

```diff
@@ -0,0 +1,112 @@
+package datadog
+
+import (
+	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceDatadogChildOrganisation() *schema.Resource {
```

Comment:
Small nit--can you please harmonize the spelling of this function name with the file name?
```suggestion
func resourceDatadogChildOrganisation() *schema.Resource {
```
```suggestion
func resourceDatadogChildOrganization() *schema.Resource {
```